resources/definitions/hellbot_magna_2_500.def.json

```diff
@@ -0,0 +1,42 @@
+{
+    "version": 2,
+    "name": "Hellbot Magna 2 500",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Hellbot Development Team",
+        "manufacturer": "Hellbot",
+        "file_formats": "text/x-gcode",
+		"platform": "Hellbot_Magna_2_500.obj",
+        "platform_texture": "Magna2_500.png",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "hellbot_magna_2_500_extruder_0"
```

Comment:
The Hellbot website claims this printer is dual-extrusion: https://hellbot.xyz/magna-2/especificaciones-magna-2-500/

Why did you include a single-extrusion version as well? Extruders can be disabled in Cura if necessary for a certain print. As far as I could tell, there is no single-extrusion edition of this printer or at least no evidence of it online.